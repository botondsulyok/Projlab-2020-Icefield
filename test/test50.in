field swimpool 2 true 0
entity esk Eskimo swimpool
item tent1 Tent esk
stats swimpool
stats esk
tent esk
stats swimpool
stats esk