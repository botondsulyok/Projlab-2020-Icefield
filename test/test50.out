Game Over
id: swimpool
type: unstable
building: plains0
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null
capacity: 2
broken: true

id: esk
type: eskimo
iceUnit: swimpool
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent1
components: null

id: swimpool
type: unstable
building: plains0
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null
capacity: 2
broken: true

id: esk
type: eskimo
iceUnit: swimpool
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: tent1
components: null

